package.json

```diff
@@ -18,17 +18,11 @@
     "react-router-dom": "^4.2.2",
     "react-scripts-ts": "2.13.0"
   },
-  "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx,json,css}": [
-      "prettier --single-quote --write",
-      "git add"
-    ]
-  },
   "scripts": {
-    "start": "react-scripts-ts start",
-    "precommit": "lint-staged",
-    "build": "react-scripts-ts build",
-    "test": "react-scripts-ts test --env=jsdom",
+    "prettier": "prettier --single-quote --parser typescript --print-width 120 --write \"src/**/*.{js,jsx,ts,tsx,json}\"",
+    "start": "npm run prettier && react-scripts-ts start",
```

Comment:
@mtho11 @aljesusg is it a correct way to embed script invocation or not? (ie. "start" invokes "prettier" script)

Reply:
lets roll with it. we can always change later

Reply:
For me is ok have this task in start